src/web/user_agent.rs

```diff
@@ -72,11 +72,18 @@ pub fn strip_ua(ua: &str) -> String {
         major = major
     )
 }
+*/
 
 /// Convert a UserAgent header into a simplified ([OsFamily], [FormFactor])
-pub fn get_device_info(ua: &str) -> (OsFamily, FormFactor) {
+pub fn get_device_info(ua: &str) -> HandlerResult<(OsFamily, FormFactor)> {
     let wresult = Parser::new().parse(ua).unwrap_or_default();
 
+    // If it's not firefox, it doesn't belong here...
+    // are there other names that could be returned? (e.g. for the iOS version)
```

Comment:
Woothee only returns "Firefox" (also "FirefoxOS" which I doubt we'll hit any more).

We already have tests for iOS/android uas below that pass, so we can kill this comment I think

Reply:
cool, wasn't sure if I was missing something.